src/main/java/com/puppycrawl/tools/checkstyle/checks/NewlineAtEndOfFileCheck.java

```diff
@@ -88,6 +88,19 @@
  * <pre>
  * &lt;module name=&quot;NewlineAtEndOfFile&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * // File ending with a new line
+ * public class Test {⤶
+ * ⤶
+ * }⤶
+ * ⤶// ok
+ *
+ * // File ending without a new line
+ * public class Test1 {⤶
+ * ⤶
+ * }⤶ // violation, the file does not end with a new line
```

Comment:
Same here. There should be no `⤶` symbol on the last line. It is assumed that the file ends immediately after the `}`' character.

Reply:
Done